packages/pdfkit/src/image/jpeg.js

```diff
@@ -43,7 +43,7 @@ class JPEG {
     this.obj = document.ref({
       Type: 'XObject',
       Subtype: 'Image',
-      BitsPerComponent: this.bits,
+      BitsPerComponent: 8,
```

Comment:
Why replace this.bits with 8, 1 byte = 8 bits :v

Reply:
this.bits is undefined, check the whole constructor its never defined.